Lib/test/test_configparser.py

```diff
@@ -915,8 +915,7 @@ def test_items(self):
         self.check_items_config([('default', '<default>'),
                                  ('getdefault', '|<default>|'),
                                  ('key', '|value|'),
-                                 ('name', 'value'),
-                                 ('value', 'value')])
+                                 ('name', 'value')])
```

Comment:
This proves this never worked. The test that added `('value', 'value')` dates back to 2010 (Python 3.2).